bazel/external/wamr_llvm.BUILD

```diff
@@ -10,30 +10,27 @@ filegroup(
 )
 
 cmake(
-    name = "llvm_13_0_1_lib",
+    name = "llvm_lib",
     cache_entries = {
         # Disable both: BUILD and INCLUDE, since some of the INCLUDE
         # targets build code instead of only generating build files.
+        # Dsiable unncessary components
```

Comment:
There is a typo (dsiable) and this comment doesn't add anything that isn't obvious from all the `offs` below.